modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -2597,16 +2595,7 @@ <P extends NGNode> P getPeer() {
     /**
      * Creates a new instance of Node.
      */
-    protected Node() {
-        //if (PerformanceTracker.isLoggingEnabled()) {
-        //    PerformanceTracker.logEvent("Node.init for [{this}, id=\"{id}\"]");
-        //}
-        updateTreeVisible(false);
-        //if (PerformanceTracker.isLoggingEnabled()) {
-        //    PerformanceTracker.logEvent("Node.postinit " +
-        //                                "for [{this}, id=\"{id}\"] finished");
-        //}
```

Comment:
Not sure if these comments are intended to be used for something. I doubt it though.